src/commands/env.rs

```diff
@@ -15,6 +15,9 @@ pub struct Env {
     #[clap(long)]
     #[clap(possible_values = AVAILABLE_SHELLS)]
     shell: Option<Box<dyn Shell>>,
+    /// Print JSON instead of shell commands.
+    #[clap(long)]
+    json: bool,
```

Comment:
We should probably make it broader, so `--output=json` is preferable. `--output=json` conflicts with `--shell=ANY` which can be declared with `#[clap(conflicts_with = "shell")]`

That being said, `--shell=...` is practically `--output=...`, right? so maybe we can leverage `--shell=` to be used as `--shell=json`? Kinda ugly naming but does not break and does not introduce another API complexity?

Reply:
Ah I didn't know about `conflicts_with`. Thanks!

Haha I _almost_ did `--shell=json`. Maybe deprecating `--shell` and replace it with `--output`? That reads well in my head. 

```sh
fnm env --output=bash
```

Then for compatbility sake just have `--shell` map to `--output`? Is there an option for that in `clap`?